dist/Renderer.js

```diff
@@ -0,0 +1,25 @@
+class Renderer {
+       renderBoard(matrix) {
+        const source = $("#matrix-template").html()
+        const template = Handlebars.compile(source)
+        const board = template({ matrix })
+        $("#matrix").empty().append(board)
+    }
+    renderScore1(score1) {
```

Comment:
maybe make one render player score method that would take two parameters `renderScore(template, score)`